src/platform/site-wide/side-nav/components/DuplicateLineLabel.js

```diff
@@ -27,7 +27,7 @@ const DuplicateLineLabel = ({ depth, item }) => {
       <a
         className={classNames({
           'va-sidenav-item-label': true,
-          'va-sidenav-item-label-duplicate': true,
+          'va-sidenav-item-label-underlined': true,
```

Comment:
Optionally, you can do `classnames('va-sidenav-item-label', 'va-sidenav-item-label-underlined', { /* conditions */ });`